booking/src/main/java/org/dcsa/conformance/standards/booking/action/Carrier_SupplyScenarioParametersAction.java

```diff
@@ -13,15 +13,17 @@ public class Carrier_SupplyScenarioParametersAction extends BookingAction {
   private CarrierScenarioParameters carrierScenarioParameters = null;
 
   private ScenarioType scenarioType;
-  public Carrier_SupplyScenarioParametersAction(String carrierPartyName, @NonNull ScenarioType scenarioType) {
+  public Carrier_SupplyScenarioParametersAction(String carrierPartyName, @NonNull ScenarioType scenarioType, boolean assignCbr) {
     super(carrierPartyName, null, null,
       switch (scenarioType) {
       case REGULAR -> "SupplyCSP";
       case REEFER -> "SupplyCSP-AR";
       case DG -> "SupplyCSP-DG";
     }, -1);
     this.scenarioType = scenarioType;
+    this.assignCbr = assignCbr;
     this.getDspConsumer().accept(getDspSupplier().get().withScenarioType(scenarioType));
+    this.getDspConsumer().accept(getDspSupplier().get().withAssignCbr(assignCbr));
   }
```

Comment:
I am surprised to see `assignCbr` in the supply scenario parameters action. I had expected a separate action that could be invoked at any time between UC1 and UC5.

Reply:
@nt-gt  George wanted to have a flag in the UC1 like assignCbr in order make it configurable. Hence I chose this way.